### PR TITLE
Feat/text field input toggle password visibility

### PR DESCRIPTION
**Pull request by julpellat**
## Description

Adding new input password for text field

-----

MerciApp_Qz3cPykkd4

### CODE REVIEW

**Comment by AlexiaAlarza** [comment]
@julpellat on dirait que l'icône est inversé. Il faut que l'oeil "visible" soit présent quand on voit le mot de passe

**Comment by LuccaIntegration** [comment]
:woman_cook: https://lucca-front.lucca.tech/PR-2434/storybook

**Comment by CCNET-iLucca** [comment]
[Tests d'interfaces](https://qa.lucca.tech/10313/ui-diff)

**Comment by LuccaIntegration** [comment]
:woman_cook: https://lucca-front.lucca.tech/PR-2434/storybook

**Comment by jeremie-lucca** [comment]
(sorry, wrong branch commit 🤦)

**Comment by CCNET-iLucca** [comment]
[Tests d'interfaces](https://qa.lucca.tech/10318/ui-diff)

File: packages/ng/forms/textfield/textfield.component.ts [hunk 1]
```diff
@@ -47,6 +49,9 @@ export class TextfieldComponent extends AbstractFieldComponent {
 	@Input({ transform: booleanAttribute })
 	hasSearchIcon = false;
 
+	@Input({ transform: booleanAttribute })
+	hasTogglePasswordVisibilityIcon = false;
```
Comment (GuillaumeNury) [inline packages/ng/forms/textfield/textfield.component.ts:53]: Do we really need this input ? We could always use visibility icon on password input

**Comment by LuccaIntegration** [comment]
:woman_cook: https://lucca-front.lucca.tech/PR-2434/storybook

**Comment by CCNET-iLucca** [comment]
[Tests d'interfaces](https://qa.lucca.tech/10416/ui-diff)

**Review by GuillaumeNury — DISMISSED** [review]
LGTM

**Comment by LuccaIntegration** [comment]
:woman_cook: https://lucca-front.lucca.tech/PR-2434/storybook

**Comment by CCNET-iLucca** [comment]
[Tests d'interfaces](https://qa.lucca.tech/10708/ui-diff)

**Review by Supamiu — APPROVED** [review]

**Review by vvalentin-lucca — APPROVED** [review]

**Review by GuillaumeNury — APPROVED** [review]

**Pull request merged**